resources/feed-template.xml

```diff
@@ -24,10 +25,12 @@
             <guid isPermaLink="false" />
             <pubDate />
             <dc:creator />
+            <media:thumbnail url="" width="" height="" />
```

Comment:
A quick question: what's the effect on the RSS item if there is no thumbnail? If the URL or any other attribute is not set how does it affect the RSS item display inside the different RSS clients?